### PR TITLE
Fixes #35345 - Host details Operating system card

### PR DESCRIPTION
**Pull request by MariaAga**
Screenshot from 2022-08-08 20-27-23
mock: https://marvelapp.com/prototype/995cc27/screen/82844194

### CODE REVIEW

File: webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/OperatingSystem/index.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+const OperatingSystemCard = ({ status, isExpandedGlobal, hostDetails }) => {
+  const {
+    architecture_name: architectureName,
+    operatingsystem_name: osName,
+    reported_data: { boot_time: bootTime } = {},
+  } = hostDetails;
+  return (
+    <CardTemplate
+      header={__('System properties')}
```
Comment (jeremylenz) [inline webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/OperatingSystem/index.js]:
```suggestion
      header={__('Operating system')}
```

**Review by jeremylenz — COMMENTED** [review]
Thanks @MariaAga!

Looks good except one thing.. :wink:

File: webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/OperatingSystem/index.js (hunk 1 above)
Comment (MariaAga) [reply webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/OperatingSystem/index.js]: Thanks, fixed :)

**Review by jeremylenz — APPROVED** [review]
LGTM! :+1:

**Pull request merged**